Homework5/task_3.py

```diff
@@ -0,0 +1,50 @@
+# Реализовать функцию get_ranges которая получает на вход
+# непустой список неповторяющихся целых чисел, отсортированных по
+# возрастанию, которая этот список “сворачивает”
+#  get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+#  get_ranges([4,7,10]) // "4,7,10"
+#  get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+
+
+def get_ranges(inp_lst):
+    outp_lst = []
+    for i in range(0, len(inp_lst)):
+        if i != 0 and i != len(inp_lst) - 1:
```

Comment:
it looks too tricky and can be solved easier
good news is that it's O(n) solution